frontend/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { useState } from "react";
+import "./SearchBar.css";
+
+const SearchBar = ({}) => {
+  return (
+    // JSX code to render component.
+    <div className="search-container">
+      <form id="seachBar">
+        <input placeholder="Look up Kudos!" name="searchBar"></input>
```

Comment:
Nit:`<input>` can be written as a self closing tag: `<input placeholder="Look up Kudos!" name="searchBar" />`